appd/request.py

```diff
@@ -732,6 +734,84 @@ def get_nodes(self, app_id=None, tier_id=None):
         path = ('/tiers/%s/nodes' % tier_id) if tier_id else '/nodes'
         return self._app_request(Nodes, path, app_id)
 
+    # RBAC requests
+
+    def _rbac_v1_request_set(self, cls, path, params=None, method='GET', query=True, use_json=True):
+        obj = self.request('/controller/api/rbac/v1' + path, params, method=method, query=query, use_json=use_json)
+        for key in obj:
```

Comment:
What is this doing? The value variable does not appear to be in scope for the return line.

Reply:
variable 'value' is in scope. 'from_json' function is expecting json response like this:

[
        {
            "id": 5,
            "name": "user1"
        },
        {
            "id": 7,
            "name": "support-user"
        },
        {
            "id": 15,
            "name": "Ash"

However, RBAC api is sending the response like below. So, I took the value and sending that instead. After this change, I could use the from_json() from the __init__ class, without any issues.

{
    "users": [
        {
            "id": 5,
            "name": "user1"
        },
        {
            "id": 7,
            "name": "support-user"
        },
        {
            "id": 15,
            "name": "Ash"